scripts/azure-linux_mac.yml

```diff
@@ -175,24 +172,26 @@ steps:
     fi
 
     # - bash: |
-    pushd $BUILD_REPOSITORY_LOCALPATH/examples/cmake_project
-    mkdir build && cd build
-    cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist .. && make
-    ./ExampleExe
-    if [[ "TILEDB_STATIC" == "ON" ]]; then
-      ./ExampleExe_static
-    fi
-    popd
-    # Build and run the PNG ingestion example.
-    if [[ "$AGENT_OS" == "Linux" ]]; then
-      # libpng (example dependency)
-      sudo apt-get install libpng-dev
-
-      pushd $BUILD_REPOSITORY_LOCALPATH/examples/png_ingestion;
-      mkdir build && cd build;
-      cmake -DCMAKE_PREFIX_PATH=$BUILD_REPOSITORY_LOCALPATH/dist .. && make;
-      ./tiledb_png $BUILD_REPOSITORY_LOCALPATH/doc/source/figures/Scarlet-Macaw-small.png /tmp/pngarray output.png;
-      popd;
+    if [[ "$TILEDB_GCS" != "ON" ]]; then
```

Comment:
Why is this disabled? Are there GCS-specific examples?

Reply:
I believe this comment is obsolete, this section has been removed. Marking resolved.